benchmarks/GeneratedHydratorBenchmark/AllPrivateClassHydrationBench.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace GeneratedHydratorBenchmark;
+
+/**
+ * @BeforeMethods({"setUp"})
+ */
+class AllPrivateClassHydrationBench extends AbstractHydrationBench
+{
+    public function setUp()
+    {
+        $this->createHydrator(AllPrivateClass::class);
+        $this->createData();
+    }
+
+    /**
+     * @Revs(100)
+     * @Iterations(200)
+     */
+    public function benchConsume()
+    {
+        $object = new AllPrivateClass();
```

Comment:
Fair enough, I will just test with a single one as a start, we'll see later if we want to improve this.